spyder/plugins/ipythonconsole/comms/tests/test_console_kernel.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
```

Comment:
Please rename this file to be `test_runcell.py` instead of `test_console_kernel.py`.